### DISCUSSION TITLE
is it possible to set fps <1

### DISCUSSION THREAD

**Discussion by vmatrix** [nateraw/stable-diffusion-videos#145] [Q&A] [answered]
i want the video transition to be slower..is it possible ?

**Comment by nateraw** [comment, marked as answer]
You could just generate the frames and then deal with them however you wish, eh? Or take the generated video and slow it down even more if you want. 

Behind the scenes, we are more or less making use of `ffmpeg`, so if you can find a way using that to do what you want, we can find a way to support it here. 😄
